src/coreclr/jit/emitarm64.cpp

```diff
@@ -17130,6 +17221,47 @@ void emitter::emitDispInsHelp(
             emitDispImm(emitGetInsSC(id), false);                           // iiii
             break;
 
+        // <Xdn>, <Pm>.<T>
+        case IF_SVE_DM_2A: // ........xx...... .......MMMMddddd -- SVE inc/dec register by predicate count
+            emitDispReg(id->idReg1(), id->idOpSize(), true);           // ddddd
+            emitDispPredicateReg(id->idReg2(), PREDICATE_NONE, false); // MMMM
```

Comment:
Once https://github.com/dotnet/runtime/pull/95918 goes in then this should be 
`emitDispPredicateReg(id->idReg2(), PREDICATE_SIZED, false);` and then don't call `emitDispArrangement()`

(This code can be fixed up later if it goes in before 95918)